### ISSUE TITLE
German translation

### ISSUE THREAD

**Issue by kludikovsky** [derhansen/sf_event_mgt#273]
There seems to be no German translation available.
I have made a suggestion for the one open item on the translation server.
It should now be made available.

**Comment by derhansen** [comment]
Hi,

I accepted your suggestion. All new language labels from version 1.3.0 do obviously not show up on the translation server. I will investigate this.

Best regards
Torben

**Comment by kludikovsky** [comment]
Hi,

 

first of all thank you this extension!

Let me know if I can be of some further help.

 

by the way:

I expererienced the same issue as https://github.com/derhansen/sf_event_mgt/issues/259 with the same resolution.

I had also initially tried to modify the constants with no effect, as well as trying to write a leading ‘1:’ in front of the directory-string.

 

Best regards,

Kurt

 


Ing. Kurt Ludikovsky

M: +43-664-30 88 770

 

 <http://www.biz-improver.at/vcards/Ludikovsky__Kurt_priv.vcf> Alle Kontaktdaten auf 
einen Klick geladen
als vCard 

 

 

Von: Torben Hansen [mailto:notifications@github.com] 
Gesendet: Samstag, 09. Jänner 2016 19:12
An: derhansen/sf_event_mgt
Cc: kludikovsky
Betreff: Re: [sf_event_mgt] German translation (#273)

 

Hi,

I accepted your suggestion. All new language labels from version 1.3.0 do obviously not show up on the translation server. I will investigate this.

Best regards
Torben
